letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -343,6 +343,14 @@ elif [ -f /etc/redhat-release ]; then
     }
     USE_PYTHON_3=1
     BOOTSTRAP_VERSION="BootstrapRpmPython3 $BOOTSTRAP_RPM_PYTHON3_VERSION"
+  # RHEL 8 uses python3.
+  elif [ "$RPM_DIST_NAME" = "rhel" -a "$RPM_DIST_VERSION" -ge 8 ]; then
```

Comment:
Is it possible to merge this condition with the previous one? It will reduce code duplication since the exact same logic is applied in case of old centos, rhel8 and fedora 29+.

Note that in my PR about migrating centos 6 users to python 3.6, I will extract this situation in a specific branch with a specific bootstrap.

Reply:
I personally don't think we should make a massive line like:
```
if [ "$RPM_DIST_NAME" = "fedora" -a "$RPM_DIST_VERSION" -ge 29 -o "$RPM_DIST_NAME" = "rhel" -a "$RPM_DIST_VERSION" -ge 8 -o "$PYVER" -eq 26 ]; then
```
(plus any parens to make that work)

We could have separate if branches that set some variable like `RPM_USE_PYTHON_3` and then a separate if branch that checks this variable and then sets the relevant bootstrap variables.

If you think we should do this, I'd encourage you to suggest it on https://github.com/certbot/certbot/pull/7240 otherwise we need to get a 3rd person in here to review any changes we make.

If you don't like my suggestion to reuse code, I think we should get on the same page ourselves before making requests in #7240.

Reply:
Yes my concern is to avoid duplication of the code, to improve readability and maintainability, but I agree that a big chunk of if condition does not reach this purpose.

I am OK with your proposition, it will make the code more explicit, while avoiding duplication of the code. 

I create a request on #7240.